opentracing-mock/src/test/java/io/opentracing/mock/MockTracerTest.java

```diff
@@ -128,7 +128,7 @@ public void testStartExplicitTimestamp() throws InterruptedException {
         {
             tracer.buildSpan("foo")
                     .withStartTimestamp(startMicros)
-                    .start()
+                    .startManual()
```

Comment:
btw. `startManual` will be probably removed in 1.0/0.31.0 I would just wait with this.